README.md

```diff
@@ -10,3 +10,7 @@ Find instructions on how to run Plausible Analytics Self Hosted [in our docs](ht
 
 We are always looking to expand on the options and setups provided here. Feel free to open an issue or PR if you feel
 something could be improved.
+
+### Upgrade guides
+
+- [Upgrading `plausible_db` (PostgreSQL)](upgrade/postgres.md)
```

Comment:
I would propose writing "from version 12 to version 14" explicitly there, as that's precisely why we need this documentation in the first place.

Reply:
The idea here is that this guide can be used to upgrade from any version, hence the `pg_dump + psql` choice. In the release notes we can specify that we recommend existing installations to be updated from 12 to 14.

Reply:
The upgrade guide should work with any PostgreSQL version >12.

We can document the supported PostgreSQL versions (12,13,14). I've added all 3 into our unit testing matrix as well: https://github.com/plausible/analytics/commit/eca71b9b60db288bfba1f366ceb4bccf8b0a4f98

Reply:
So I think to address the original concern, I would propose adding something like:

`Supported PostgreSQL versions are 12,13, and 14`